src/app/modules/angular-slickgrid/styles/slick-bootstrap.scss

```diff
@@ -436,4 +442,40 @@
   .slick-viewport-bottom.slick-viewport-right {
     overflow-y: $frozen-overflow-right !important;
   }
+  .input-group {
+    > :not(:first-child):not(.dropdown-menu):not(.valid-tooltip):not(.valid-feedback):not(.invalid-tooltip):not(.invalid-feedback) {
+      margin-left: 0;
+      &.ui-slider-horizontal {
+        margin-left: 10px;
+      }
+    }
+
+    .input-group-addon {
+      .input-group-text {
+        height: 100%;
+      }
+      &.input-group-append {
+        .input-group-text {
+          margin-left: -1px;
+          border-top-left-radius: 0;
+          border-bottom-left-radius: 0;
+        }
+      }
+      &.input-group-prepend {
+        .input-group-text {
+          margin-right: -1px;
+          border-top-right-radius: 0;
+          border-bottom-right-radius: 0;
+        }
+      }
+    }
+  }
+}
+
+.slick-pagination {
+  height: 40px;
+
+  .slick-pagination-nav .pagination .page-item a[class*="icon-seek-"] {
+    line-height: 1rem;
```

Comment:
since I use `px` everywhere I think it might be better to add these two (`height` and `line-height`) as SASS variables (in the `_variables.scss` file), there's a few variables starting with `$pagination-` and so if you could add them there.

Reply:
will do

Reply:
Addressed by https://github.com/ghiscoding/Angular-Slickgrid/pull/657/commits/731fc0214a376f3872b7181a7d19a99a6673e5f0